python/nav/web/templates/portadmin/portlist.html

```diff
@@ -3,16 +3,25 @@
 
 {{ aliastemplate }}
 
+{% if messages %}
+{% include 'portadmin/_messages.html' with hx_swap_oob=True %}
+{% endif %}
 
-<div id="portadmin-wrapper" class="faketable">
-
+<div id="portadmin-wrapper" class="faketable{% if loading %} loading{% endif %}">
   <div class="caption">
     <a href="{{ netbox.get_absolute_url }}"
        title="Go to Ip Device Info for {{ netbox }}">
       {{ netbox.get_short_sysname }}
     </a>
     {% if handlertype %}({{ handlertype }}){% endif %}
-
+    {% if loading %}
+    <div class="loading-indicator right">
+        <span>
+            <img src="/static/images/select2/select2-spinner.gif" alt="Loading spinner" />
+        </span>
+        Loading live data from devices
```

Comment:
There's only one device involved in the communication:

```suggestion
        Loading live data from device
```